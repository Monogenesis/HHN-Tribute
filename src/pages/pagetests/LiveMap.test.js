import React from "react";
import renderer from "react-test-renderer";
import Livemap from "../LiveMap.js";

test("LiveMap Page renders correctly", () => {
  const tree = renderer.create(<Livemap></Livemap>).toJSON();
  expect(tree).toMatchSnapshot();
});
