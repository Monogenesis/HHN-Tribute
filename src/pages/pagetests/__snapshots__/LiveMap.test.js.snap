// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiveMap Page renders correctly 1`] = `
<body>
  <div>
    <h1
      className="LivemapTitle"
    >
      HeatMap
    </h1>
  </div>
  <div
    id="heatMap"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": "90vh",
        "justifyContent": "center",
      }
    }
  >
    <div>
      <div>
        <div
          className="selectTitle"
        >
          <span>
            <h2>
              Auswahl:
            </h2>
          </span>
        </div>
        <div
          className="selectDiv"
        >
          <select
            className="selectSection"
            defaultValue={
              Object {
                "label": "Woher sind unsere Helden",
                "value": "userMap",
              }
            }
            onChange={[Function]}
          >
            <option
              value="userMap"
            >
              Woher unsere Helden kommen
            </option>
            <option
              value="feverMap"
            >
              Erhöhte Körpertemperatur
            </option>
          </select>
        </div>
        <div
          className="mapDiv"
        >
          <div
            id="map"
            style={
              Object {
                "height": 700,
                "width": 960,
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
